eslint.config.js

```diff
@@ -1,6 +1,6 @@
-import js from "@eslint/js";
-import eslintConfigPrettier from "eslint-config-prettier";
-import gts from "./node_modules/gts/build/src/index.js";
+import js from '@eslint/js';
```

Comment:
I think you haven't made any changes to this file anymore, other than running Prettier over it. If that's the case, you should revert it from this PR.